mentors/README.md

```diff
@@ -1,78 +1,127 @@
 
+## How to become a mentor?
 
+Short answer: come up with a project idea that is suitable for a mentorship and inform the CNCF mentoring group by sending a PR!
 
-## Success Stories
-
-
-
-## FAQs
+Long answer:
+1. Watch out for the announcements: CNCF mentoring programs are announced on CNCF Twitter ([@CloudNativeFdn](https://twitter.com/CloudNativeFdn)), and CNCF Slack ([#mentoring](https://cloud-native.slack.com/archives/CGPK98JNQ)). You can also watch the [announcements](https://github.com/cncf/mentoring/discussions/categories/announcements) on CNCF mentoring repository.
+2. Find a project idea: Either come up with a new mentorship project idea, or find an existing mentorship project idea that you would like to mentor for. Definition of what is a good project idea varies from program to program. For example, while LFX Mentorship project ideas are more focused on coding, Google Season of Docs project ideas are more focused on documentation.
+3. Submit your project idea: Submit your project idea to the [CNCF mentoring repository](https://github.com/cncf/mentoring). You can use the [project idea template](https://github.com/cncf/mentoring/blob/main/PROJECT_IDEA_TEMPLATE.md). There will be information on how to submit your project idea in the announcement that will be sent for the program.
+4. Review mentee profiles and proposals: Once the mentee profiles and proposals are submitted for your project idea, you will be able to review them. Program administrators will provide you with more information on how to review the proposals and the main criteria for acceptance will be your feedback.
+5. Mentor the mentees: Once the mentees are accepted into the program, you will be able to start mentoring them! 
 
+The processes may vary from program to program, however you will be informed at every step about what to do and how to do it by the program administrators, including a timeline!
 
+## Programs
 
+For the up-to-date list of programs, please check the [CNCF mentoring repository](https://github.com/cncf/mentoring).
 
----
+## Example mentoring timeline
 
-Mentees - Frequently Asked Questions (FAQs)
-FAQs: New Mentees and Contributors
-General Questions
+* GSoC at CNCF 2023 is [announced](https://github.com/cncf/mentoring/discussions/848)
+  * Mentors send their project idea PRs ([example](https://github.com/cncf/mentoring/pull/810))
+* Mentee applications start ([announcement](https://github.com/cncf/mentoring/discussions/892))
+  * Mentee candidates interact with the community and the mentors
+  * Mentee candidates submit their proposals to the GSoC platform
+*  Mentee applications close ([announcement](https://github.com/cncf/mentoring/discussions/918))
+  * Mentors review the proposals and provide feedback
+* Accepted mentees are [announced](https://github.com/cncf/mentoring/discussions/954)
+  * Mentees bond with the community and the mentors
+  * Mentors and mentees start working on their projects
+* TBA: (we're in the middle of the program)
```

Comment:
I am curious about your opinions on this section. What I wanted to provide is a realistic view of the "paperwork" (submiting project idea, reviewing mentee proposals, etc).